src/System.Device.Gpio/System/Device/Gpio/Drivers/Windows10Driver.Linux.cs

```diff
@@ -1,73 +1,73 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Whar really changed on this file? Was it only EOL chars?

Reply:
Yes, there were a few EOL prompts when opening these files in VS.  There are a few others throughout projects, so you might see those from other PRs I may have.